packages/adapter-cloudflare-workers/README.md

```diff
@@ -33,9 +33,19 @@ export default {
 
 **You will need [Wrangler](https://developers.cloudflare.com/workers/cli-wrangler/install-update) installed on your system**
 
-This adapter expects to find a [wrangler.toml](https://developers.cloudflare.com/workers/platform/sites/configuration) file in the project root. It will determine where to write static assets and the worker based on the `site.bucket` and `site.entry-point` settings.
+This adapter expects to find a [wrangler.toml](https://developers.cloudflare.com/workers/platform/sites/configuration) file in the project root. It will determine where to write static assets and the worker based on the `site.bucket` and `build.upload` settings. These values must be set to the following:
 
-Generate this file using `wrangler` from your project directory
+```toml
+[build.upload]
```

Comment:
This approach is more opinionated than the last one, which IMO is a better developer experience. Given `adapter-cloudflare` builds to `.svelte-kit/cloudflare` and requires the user to set it to that path in Pages, I don't think this leaking any more details. That said, I can revert it if the old style, user-provided config is preferred.